aws-resources/src/main/java/io/opentelemetry/contrib/aws/resource/EcsResource.java

```diff
@@ -229,8 +235,22 @@ static void parseResponse(
       }
     }
 
-    getRegion(arn).ifPresent(region -> attrBuilders.put(CLOUD_REGION, region));
-    getAccountId(arn).ifPresent(accountId -> attrBuilders.put(CLOUD_ACCOUNT_ID, accountId));
+    String region = getRegion(arn).orElse(null);
+    String account = getAccountId(arn).orElse(null);
+    if (region != null) {
+      attrBuilders.put(CLOUD_REGION, region);
+    }
+    if (account != null) {
+      attrBuilders.put(CLOUD_ACCOUNT_ID, account);
+    }
+    if (cluster != null) {
+      if (cluster.contains(":")) {
```

Comment:
nb: should we do a more comprehensive check to be sure the `cluster` is in ARN format? Something like
```
cluster.startsWith("arn:aws:ecs:")
```

Reply:
> Cluster name must be 1 to 255 characters. Valid characters are a-z, A-Z, 0-9, hyphens (-), and underscores (_).

Checking `:` and `arn:aws:ecs:` gives us the same result.